src/components/Stepper/Stepper.tsx

```diff
@@ -522,6 +522,7 @@ export const Stepper: FC<StepperProps> = React.forwardRef(
                   className={styles.stepsContainer}
                   onScroll={handleScroll}
                   ref={stepsContainerRef}
+                  tabIndex={0}
```

Comment:
seems like you could do this dynamically such that it isn't in the tab order when the scroll isn't present. something like (please test)

```suggestion
                  tabIndex={mergedScrollable ? 0 : null}
```

Reply:
Updated tabindex attribute condition as suggested.